ci/scripts/harbor/harbor_utils.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+HEADER="application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"
+BASE_URL="registry.nordix.org"
+PROJECT="projects/metal3"
+HARBOR_API="https://${BASE_URL}/api/v2.0/${PROJECT}/repositories"
+
+# DRY_RUN argument is present in all of the  functions that call the Harbor API.
+# The default value for DRY_RUN is true thus unless it is manually set to false
+# the function won't call the API instead it will print out the intended action.
+
+# RESULT_NUM is a common argument and it specifies the maximum number of top level
+# JSON objects in the JSON array that is returned after a API call.
+
+# NOTE: Tags are not unique identifiers but digests are!
+
+# NOTE: digests in the cleanup exclusion list file have to be in the following format
+# sha256:<digest_number>
+
+# Cleans a OCI repository that is specified as the first argument.
+# It is required to supply a file that contains a clear text list of the
+# digests of the artifacts that are excluded from the cleanup.
+# There is also an argument named RETENTION that specifiex the number of the
+# latest artifacts that won't be deleted.
+harbor_clean_OCI_repository(){
```

Comment:
```suggestion
harbor_clean_oci_repository(){
```
nit and personal choice, feel free to keep is you wish

Reply:
I would leave it like this as it is an acronym of a standard.